RecoLocalCalo/HcalRecAlgos/interface/PulseShapeFunctor.h

```diff
@@ -64,23 +54,34 @@ namespace FitterFuncs {
     inline double doublePulseShapeFunc(const double *x) { return EvalPulseM2(x, 5); }
     inline double triplePulseShapeFunc(const double *x) { return EvalPulseM2(x, 7); }
 
-    void getPulseShape(std::array<double, HcalConst::maxSamples> &fillPulseShape) { fillPulseShape = pulse_shape_; }
+    void getPulseShape(std::array<double, hcal::constants::maxSamples> &fillPulseShape) {
+      fillPulseShape = pulse_shape_;
+    }
+
+    // getters
+    inline std::vector<float> const &get_acc25nsVec() const { return acc25nsVec; }
```

Comment:
get rid of `get_` perhaps?

Reply:
removing `get_` create conflicts with with a previous declaration
     std::vector<float> acc25nsVec

Reply:
data members should also have their names distinguished, e.g., with a trailing `_`.
so,

```suggestion
    inline std::vector<float> const &acc25nsVec() const { return acc25nsVec_; }
```